roles/st2mistral/tasks/config.yml

```diff
@@ -9,6 +9,8 @@
   with_dict: "{{ _conf_options }}"
   loop_control:
     loop_var: _conf_option
+    # prevent logging passwords, auth URIs, and secrets
+    label: "conf | {{ _conf_section_name }}.{{ _conf_option.key }}"
```

Comment:
I'm not quite sure if we should really hide _all_ the `conf` settings in Ansible debugging to mask several secret fields.

The places we've added `no_log` are where tokens could leak during the Travis CI builds.

Reply:
Yeah, I was concerned about doing that too. This was easy though.

Do you know if `no_log` applies on the loop as a whole, or on each item of the loop? If it does, then we could check and see if a particular setting is in a blacklist, and then `no_log` it.

Note that merely changing the label does not prevent logging the output if `--diff` or a higher verbosity is enabled. `no_log` is a better solution, but I didn't want to enable no_log on all of those, because I like to use `--diff`.